azure-pipelines.release-vnext.yml

```diff
@@ -0,0 +1,82 @@
+pr: none
+trigger: none
+
+# Customize build number to include major version
+# Example: v9_20201022.1
+name: 'v9_$(Date:yyyyMMdd)$(Rev:.r)'
+
+variables:
+  - group: 'Github and NPM secrets'
+  - template: .devops/templates/variables.yml
+    parameters:
+      skipComponentGovernanceDetection: false
+  - name: release.vnext # Used to scope beachball to release only vnext packages
+    value: true
+
+pool: 'Self Host Ubuntu'
+
+# schedules:
```

Comment:
Is the intent to uncomment this once we're ready to start running it?

Also is there any way this can be made to do nightly releases as well, or are we handling that separately and/or ignoring it for now?

Reply:
we can start running it manually initially for easier follow-up and validation while we figure out an appropriate schedule.

I'm going to continue to work on nightly releases since it's not strictly blocking beta release